main.nf

```diff
@@ -168,14 +171,40 @@ process get_software_versions {
     """
 }
 
+/*
+ * Run pre-checks of pipeline parameters that would otherwise cause the
```

Comment:
is it possible to add a parameter to directly list the output of available methods, versions?